### PR TITLE
SVG Demo UI Update 

### PR DESCRIPTION
**Pull request by ststimac**
@molant @melanierichards this is ready for initial review at your leisure. There are a few items that I know will need to be removed. There are some inline styles for display: none; that directly tie to the JS file and I'm not sure what to edit. If I try and apply a class to the element, the action to show the element no longer works. 

There are also two "!important" rules that I want to remove and I'm sure they can be, it's just a matter of finding the correct bit in the Javascript to remove. Previously the buttons in the sidebar all had a preview of the gradient directions and it was overwhelming. The buttons as well need an active state, and I again think that is being controlled in the Javascript. 

Anton, the other thing I would like to do is separate out the hex code from the color swatch so that there are two different input options, either by color picker which populates the hex code, or if you enter the hex code it changes the color swatch. I tried and tried to get this to work correctly but was having difficulty with the adding that third column, and adding new rows as you add new color stops. 
I updated all the Javascript ids to camel case, and cleaned up the CSS too.

### CODE REVIEW

File: svgradientbackgroundmaker/images/btn_delete.svg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 20.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 11 11" enable-background="new 0 0 11 11" xml:space="preserve">
+<g>
+	<g>
+		<polygon fill="#E81123" points="8.9,10.7 5.5,7.4 2.1,10.7 0.3,8.9 3.6,5.5 0.3,2.1 2.1,0.3 5.5,3.6 8.9,0.3 10.7,2.1 7.4,5.5 
+			10.7,8.9 		"/>
```
Comment (molant) [inline svgradientbackgroundmaker/images/btn_delete.svg]: Empty spaces here, have you run it through svggo?

**Review by molant — CHANGES_REQUESTED** [review]
I think we are not going to waste time improving the old JS. As long as the demo looks better I'm fine with it. @melanierichards what do you think?
@ststimac uou also have to add a config.json file in the root:

```
{
  isHtml: true
}
```

so we don't have any problem with other styles.

Nice job!

**Comment by ststimac** [comment]
@molant added the json file. 

I'm also fine with leaving everything the way it is (JavaScript wise) and not bothering with separating the hexcode and swatch.

**Comment by melanierichards** [comment]
@ststimac whew this looks so much nicer now. Only comment is it doesn't seem like the Montserrat font is being loaded in the CSS, probably just your local copy (it's falling back to Arial on my machine). Besides fixing that, we might also want to put Segoe UI in the font stack after Montserrat.

**Comment by ststimac** [comment]
OK, I'll check that and add Segoe UI.

**Comment by molant** [comment]
@ststimac can you try adding the following after the rest of scripts?

```
<script type="text/javascript" src="https://msedgecdn.azurewebsites.net/scripts/demo-header.js"></script>
```

**Comment by ststimac** [comment]
@molant that has been added.

**Review by molant — APPROVED** [review]

**Pull request merged**